Makefile

```diff
@@ -15,7 +15,7 @@ ci-build-misc: print-go-version bins ci-update-tools shell-check copyright-check
 clean: clean-bins clean-test-results
 
 # Recompile proto files.
-proto: clean-proto install-proto-submodule buf-lint api-linter protoc fix-proto-path goimports-proto proto-mocks copyright-proto
+proto: clean-proto buf-lint api-linter protoc fix-proto-path goimports-proto proto-mocks copyright-proto
```

Comment:
is there a way to make it init only if not init-ed, but otherwise leave it alone? maybe `git submodule init` does that? my only concern is that it's nice if running `make proto` in a fresh clone just works. (if that's too hard then it's fine)

Reply:
I think requiring `git submodule init` on a fresh clone is fine and is fairly common practice.
Note that there's another submodule (https://github.com/temporalio/dashboards) that we don't automatically init either.